client/src/components/PageComponents/CalendarComponent/CalendarComponent.tsx

```diff
@@ -3,131 +3,182 @@ import Calendar from "react-calendar";
 import "react-calendar/dist/Calendar.css";
 import { BsCalendar4 } from "react-icons/bs";
 import "./Calendar.css";
-import { calendarContainer, iconArea, iconStyle, locationChangeText, loggoArea, loggoAreaRight, monthText, seeAllSchedules, textArea, todo, todolistContainer, todolistDetail, todolistPannel, todoTitle } from "./CalendarComponent.css";
+import {
+  calendarContainer,
+  iconArea,
+  iconStyle,
+  locationChangeText,
+  loggoArea,
+  loggoAreaRight,
+  monthText,
+  seeAllSchedules,
+  textArea,
+  todo,
+  todolistContainer,
+  todolistDetail,
+  todolistPannel,
+  todoTitle,
+} from "./CalendarComponent.css";
+import { useNavigate } from "react-router-dom";
 
-const CalendarComponent : FC = () => {
-    const [value, setValue] = useState(new Date());
-    const [isChecked, setIsChecked] = useState(false);
+const CalendarComponent: FC = () => {
```

Comment:
이제는 FC 타입을 사용하지 않는게 더 일반적입니다~
```suggestion
const CalendarComponent = () => {
```